srcpkgs/cagebreak/template

```diff
@@ -1,20 +1,20 @@
 # Template file for 'cagebreak'
 pkgname=cagebreak
-version=2.2.1
+version=2.2.3
 revision=1
 build_style=meson
 configure_args="--buildtype=release -Dman-pages=true $(vopt_bool xwayland xwayland)"
 conf_files="/etc/xdg/cagebreak/config"
 hostmakedepends="pkg-config wayland-devel scdoc"
-makedepends="wlroots-devel cairo-devel pango-devel libevdev-devel"
+makedepends="wlroots-devel libdisplay-info-devel cairo-devel pango-devel libevdev-devel"
```

Comment:
```suggestion
makedepends="wlroots0.17-devel cairo-devel pango-devel libevdev-devel"
```
Moving the `libdisplay-info-devel` into dependencies for `wlroots0.17-devel` takes care of the requirement here, and I think every compositor template should depend directly on the specific wlroots package that it requires rather than allowing future updates to the metapackage to break builds.